Modules/Module_Processing/Module_ASL/xASL_wrp_CreateAnalysisMask.m

```diff
@@ -206,11 +214,18 @@ function xASL_wrp_CreateAnalysisMask(x)
      FinalThreshold = (ThresholdIntensity+SortedInt)/2; % average of 2 thresholds
      FinalMask = MixedIm>FinalThreshold;
 
-     MaskSusceptibility = BrainMask;
-     MaskSusceptibility(MaskSuscept) = FinalMask(MaskSuscept);
+     % %% Brief explanation what these masks here mean:
```

Comment:
@jan-petr Here, you had a comment that these masks where unclear. So I added some comments to clarify them.

For some strange reason, this conversation disappeared...

Reply:
I guess that the issue is that you have to check the PR or the commit summary, not the individual comments. I see the orgiinal comment and I have resolved it already.